warehouse/api.go

```diff
@@ -122,6 +123,15 @@ func InitWarehouseAPI(dbHandle *sql.DB, log logger.Logger) error {
 	if region := config.GetString("region", ""); region != "" {
 		labels["region"] = region
 	}
+
+	client := cpclient.NewInternalClient(
```

Comment:
```suggestion
	client := cpclient.NewInternalClientWithCache(
```
Any specific reason why we are not using cached client?